src/platform/EFR32/efr32-chip-mbedtls-config.h

```diff
@@ -233,16 +232,24 @@ typedef void mbedtls_ecp_restart_ctx;
 #if OPENTHREAD_CONFIG_ECDSA_ENABLE
 #endif
 
-#define MBEDTLS_MPI_WINDOW_SIZE 1       /**< Maximum windows size used. */
-#define MBEDTLS_MPI_MAX_SIZE 32         /**< Maximum number of bytes for usable MPIs. */
-#define MBEDTLS_ECP_MAX_BITS 256        /**< Maximum bit size of groups */
-#define MBEDTLS_ECP_WINDOW_SIZE 2       /**< Maximum window size used */
+#define MBEDTLS_MPI_WINDOW_SIZE 1 /**< Maximum windows size used. */
+#define MBEDTLS_MPI_MAX_SIZE 32 /**< Maximum number of bytes for usable MPIs. */
+#define MBEDTLS_ECP_MAX_BITS 256 /**< Maximum bit size of groups */
+#define MBEDTLS_ECP_WINDOW_SIZE 2 /**< Maximum window size used */
 #define MBEDTLS_ECP_FIXED_POINT_OPTIM 0 /**< Enable fixed-point speed-up */
-#define MBEDTLS_ENTROPY_MAX_SOURCES 1   /**< Maximum number of sources supported */
+#define MBEDTLS_ENTROPY_MAX_SOURCES 1 /**< Maximum number of sources supported */
 
 // #if OPENTHREAD_CONFIG_MULTIPLE_INSTANCE_ENABLE
-#define MBEDTLS_PLATFORM_STD_CALLOC calloc /**< Default allocator to use, can be undefined */
-#define MBEDTLS_PLATFORM_STD_FREE free     /**< Default free to use, can be undefined */
+#define MBEDTLS_PLATFORM_STD_CALLOC CHIPPlatformMemoryCalloc /**< Default allocator to use, can be undefined */
+#define MBEDTLS_PLATFORM_STD_FREE CHIPPlatformMemoryFree /**< Default free to use, can be undefined */
+#ifdef __cplusplus
+extern "C" {
+#endif
+extern void * CHIPPlatformMemoryCalloc(size_t, size_t);
+extern void CHIPPlatformMemoryFree(void *);
+#ifdef __cplusplus
+} // extern "C"
```

Comment:
@mspang I don't want to do this but I can't seem to figure out how to get GN to let me include the header without creating a dependency loop (platform → support → config → platform).

Reply:
It's possible to create a source_set that just includes the platform malloc headers if necessary. It need not include the source files. In fact headers can be in multiple source_sets.